server/service/src/sync/translations/mod.rs

```diff
@@ -71,6 +72,36 @@ pub(crate) fn all_translators() -> SyncTranslators {
     ]
 }
 
+/// Calculates the integration order based on the PullDependencies in the SyncTranslators
+pub(crate) fn pull_integration_order(translators: &SyncTranslators) -> Vec<&'static str> {
+    let mut ts = TopologicalSort::<&str>::new();
```

Comment:
Nice, `TopologicalSort` makes it soo much easier

Reply:
The API of this lib is a bit unintuitive though...

Reply:
Whooaaa cool so this sorts the table in a top down references sort of fashion?

Reply:
Yes in every step it removes/pops all entries without dependencies till there are no more entries. (In case it can't finish there are circular dependencies...)